composer.json

```diff
@@ -25,7 +27,10 @@
         "ext-mbstring": "*",
         "ext-curl": "*",
         "composer/ca-bundle": "~1.0",
-        "symfony/options-resolver": "~2.7|~3.0"
+        "symfony/options-resolver": "~2.7|~3.0",
+        "php-http/httplug": "~1.1",
+        "php-http/curl-client": "~1.7",
```

Comment:
I've updated the README to explain how to install and use this library based on the great work found in the link you provided. If you have any idea on how to improve docs I'm open to suggestion not being a naive English speaker